src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -214,29 +166,75 @@ export class MeetingManager implements AudioVideoObserver {
     this.audioVideoObservers = {};
   }
 
-  async join({
-    meetingInfo,
-    attendeeInfo,
-    deviceLabels = DeviceLabels.AudioAndVideo,
-    eventReporter,
-    meetingManagerConfig,
-  }: MeetingJoinData) {
-    if (meetingManagerConfig) {
-      this.meetingManagerConfig = meetingManagerConfig;
-    }
-    this.configuration = new MeetingSessionConfiguration(
-      meetingInfo,
-      attendeeInfo
+  async join(meetingJoinData: MeetingJoinData): Promise<void> {
```

Comment:
We want this API to look like this:
```
async join(meetingSessionConfiguration: MeetingSessionConfiguration, options?: MeetingJoinOptions) {
..do the parsing for the MeetingJoinOptions as you are currently.
}
```

Sorry, I missed this!

Reply:
Done